specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyListByDatabase.json

```diff
@@ -18,8 +18,7 @@
               "weeklyRetention": "P1M",
               "monthlyRetention": "P1Y",
               "yearlyRetention": "P5Y",
-              "weekOfYear": 5,
-              "makeBackupsImmutable": true
```

Comment:
Just realize I missed testing this one, but I would imagine the return value would be same as get policy, please keep  makeBackupsImmutable